apps/GNOME/uninstall

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+purge_packages gnome gnome-core gnome-software gnome-shell || exit 1
```

Comment:
still wrong... not going to say why as I have already said it multiple times now